### ISSUE TITLE
Confused about the split of dataset in code/paper.

### ISSUE THREAD

**Issue by LeopoldACC** [Soumyabrata/CloudSegNet#3]
Thanks for answering in the [https://github.com/Soumyabrata/CloudSegNet/issues/2#issue-1207608271](https://github.com/Soumyabrata/CloudSegNet/issues/2#issue-1207608271),but the logic of the code show that the list to be shuffled is the all augmented data list,which may lead to use the variants of that particular training image in the testing set or validation set and that is not the same as you said.
And the description of dataset in Three part in your paper write as below:
```bash
We use this augmented composite SWINySeg dataset of 6768 images to train the CloudSegNet model. 
We perform a random sampling on this composite dataset, and divide the training and testing sets in 
the ratio of 80 : 20% respectively.The trend of binary cross-entropy loss for training and testing sets are shown in Fig. 2.
We observe that the loss saturates after a few thousand iterations, and the model exhibits comparable loss 
performance for both training and testing sets. We choose the CloudSegNet model with the lowest validation 
loss for our subsequent experiments.
```
I'm confused that the code and the paper both show that the logic is use the variants of that particular training image in the testing set or validation set.
But your answer is not to keep the same logic with your code as below,which confused me a lot.
```
Thanks for the interest. It is advisable that if you use any particular image in your training dataset, I'd suggest not to use the variants of that particular training image in the testing set or validation set. All variants of that particular training image should be used in the same training set.
```
Because compare different semantic segmentation method in different setting of dataset is not fair.What confused me is that if I don't keep the same with your code do,it is not fair to other method.
